docs/index.rst

```diff
@@ -101,8 +104,15 @@ Typing PEPs
 * `PEP 613 <https://www.python.org/dev/peps/pep-0613/>`_, ``TypeAlias``
 * `PEP 646 <https://www.python.org/dev/peps/pep-0646/>`_, variadic generics and ``TypeVarTuple``
 * `PEP 647 <https://www.python.org/dev/peps/pep-0647/>`_, ``TypeGuard``
+* `PEP 649 <https://www.python.org/dev/peps/pep-0649/>`_ (draft), ``from __future__ import co_annotations``
 * `PEP 655 <https://www.python.org/dev/peps/pep-0655/>`_, ``Required`` and ``NotRequired``
 * `PEP 673 <https://www.python.org/dev/peps/pep-0673/>`_, ``Self``
 * `PEP 675 <https://www.python.org/dev/peps/pep-0675/>`_, ``LiteralString``
-* `PEP 677 <https://www.python.org/dev/peps/pep-0677/>`_ (rejected), callable type syntax
-* `PEP 681 <https://www.python.org/dev/peps/pep-0681/>`_ (draft), ``@dataclass_transform()``
+* `PEP 677 <https://www.python.org/dev/peps/pep-0677/>`_ (rejected), ``(int, str) -> bool`` callable type syntax
+* `PEP 681 <https://www.python.org/dev/peps/pep-0681/>`_ ``@dataclass_transform()``
+* `PEP 688 <https://www.python.org/dev/peps/pep-0688/>`_ (draft), ``Buffer``
+* `PEP 692 <https://www.python.org/dev/peps/pep-0692/>`_ (draft), ``Unpack[TypedDict]`` for ``**kwargs``
```

Comment:
This one is accepted but we need to do some implementing I think.

Reply:
Oh great, I thought I'd seen something like that but the PEP still said draft. I'll go ahead and remove the draft here to save some bother.

I think both mypy and pyright fully support it, I can take a look at what typing_extensions / typing need